scripts/mods/appliedenergistics_singularity.zs

```diff
@@ -0,0 +1,101 @@
+#modloaded zenutils appliedenergistics2
+#sideonly client
+
+import crafttweaker.text.ITextComponent;
+
+import mods.zenutils.StaticString;
+import mods.zenutils.StringList;
+
+import scripts.jei.crafting_hints;
+
+// Clear singularity tags
+recipes.addHiddenShapeless(
+  'singularity_tag_clearing',
+  <appliedenergistics2:material:48> * 2,
+  [<ore:singularityEntangled>, <ore:singularityEntangled>]
+);
```

Comment:
This recipe would not work on servers because `#sideonly client`

Reply:
yeah exactly, it seems that I forgot to sync my repo with local repo